scripts/build-css

```diff
@@ -2,9 +2,8 @@
 
 set -e # exit when error
 
-VERSION=$(json version < package.json)
-NAME='docsearch.js'
-LICENSE="/*! ${NAME} ${VERSION:-UNRELEASED} | © Algolia | github.com/algolia/docsearch.js */"
+NAME='docsearch'
```

Comment:
The example in `dev` uses symlinks that points to the css files in `dist`, so I need to run `npm run build:css` when I run `npm run dev`